manifests/server.pp

```diff
@@ -60,12 +60,19 @@
 
   validate_bool($directoryenvs)
 
+  $service = $servertype ? {
+    'passenger'    => 'httpd',
```

Comment:
the puppetlabs/apache module calls the service resource httpd, see https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/service.pp#L41